updates/0.0.1-admins.js

```diff
@@ -7,11 +7,11 @@
  * module.exports = function(done) { ... }
  */
 
-// exports.create = {
-// 	User: [
-// 		{ 'name.first': 'Admin', 'name.last': 'User', 'email': 'fake@fake.com', 'password': 'fake', 'isAdmin': true },
-// 	],
-// };
+exports.create = {
+	User: [
+		{ 'name.first': 'Admin', 'name.last': 'User', 'email': 'saurabh@nyaaya.in', 'password': 'admin', 'isAdmin': true },
```

Comment:
remove your email and opt for something like admin or contact

Reply:
has the provision of adding more users; rather than admin shouldn't each of us have individual users? Helps in better logging?

Reply:
resolved.